display_list/display_list_builder.cc

```diff
@@ -291,6 +291,48 @@ void DisplayListBuilder::onSetMaskFilter(const DlMaskFilter* filter) {
   }
 }
 
+void DisplayListBuilder::setAttributesFromDlPaint(
+    const DlPaint& paint,
+    const DisplayListAttributeFlags flags) {
+  if (flags.applies_anti_alias()) {
+    setAntiAlias(paint.isAntiAlias());
+  }
+  if (flags.applies_dither()) {
+    setDither(paint.isDither());
+  }
+  if (flags.applies_alpha_or_color()) {
+    setColor(paint.getColor().argb);
+  }
+  if (flags.applies_blend()) {
+    setBlendMode(paint.getBlendMode());
+  }
+  if (flags.applies_style()) {
+    setStyle(ToSk(paint.getDrawStyle()));
+  }
+  if (flags.is_stroked(ToSk(paint.getDrawStyle()))) {
+    setStrokeWidth(paint.getStrokeWidth());
+    setStrokeMiter(paint.getStrokeMiter());
+    setStrokeCap(ToSk(paint.getStrokeCap()));
+    setStrokeJoin(ToSk(paint.getStrokeJoin()));
+  }
+  if (flags.applies_shader()) {
+    setColorSource(paint.getColorSource().get());
+  }
+  if (flags.applies_color_filter()) {
+    setInvertColors(paint.isInvertColors());
+    setColorFilter(paint.getColorFilter().get());
+  }
+  if (flags.applies_image_filter()) {
+    setImageFilter(paint.getImageFilter().get());
+  }
+  // if (flags.applies_path_effect()) {
+  //   setPathEffect(sk_ref_sp(paint.getPathEffect()));
+  // }
```

Comment:
Commented out code.

We need this right? Text can do it?

Reply:
I'm waiting for the DlPathEffect PR to land which will happen any day/hour now. It's just waiting for the developer to rebase one of their PRs.

Reply:
Can you add a TODO with a link to the issue/PR?